app.midterm.MidtermDesktop.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Should be upstreamed along  with the metainfo, the icons as they are not Flathub specific.

Reply:
I moved to the icons and the desktop file to the deb package. Thanks